src/EventStore.Core.Tests/ClientAPI/catchup_filtered_subscription.cs

```diff
@@ -95,12 +96,7 @@ public class catchup_filtered_subscription<TLogFormat, TStreamId> : Specificatio
 				Assert.Fail("Checkpoint reached not called enough times within time limit.");
 			}
 
-			if (LogFormatHelper<TLogFormat, TStreamId>.IsV2) {
-				Assert.AreEqual(10, eventsSeen);
-			} else {
-				// accounting for stream records
-				Assert.AreEqual(7, eventsSeen);
-			}
+			Assert.AreEqual(10, eventsSeen);
```

Comment:
is it known why this was flaky the was it was before?

Reply:
The subscription might still continue after reaching the count down resulting in errors like:
```
  Error Message:
     Expected: 7
  But was:  9
```

Reply:
ooh it had processed enough events to trigger the checkpoint but not all of them. got it :+1: